docs/sources/guides/whats-new-in-v5-1.md

```diff
@@ -0,0 +1,111 @@
+
+* [Improved docker image]({{< relref "#improved-docker-image-breaking-change" >}}) with a breaking change!
+* [Improved workflow for provisioned dashboards]({{< relref "#improved-workflow-for-provisioned-dashboards" >}})
+* [New template variable interpolation syntax]({{< relref "#new-template-variable-interpolation-syntax" >}})
+* [Alerting]({{< relref "#alerting" >}})
+* [Datasources]({{< relref "#datasources" >}}) Enhancements for Prometheus, Cloudwatch, PostgreSQL and MySQL. This release also adds support for [Microsft SQL Server]({{< relref "#microsoft-sql-server" >}}) as metric & table datasource!
+* [Panels]({{< relref "#panels" >}}) Enhancements for Graph, Table and Singlestat panels.
+* [UX improvements]({{< relref "#ux-improvements" >}})
+
+## Improved docker image (breaking change)
+
+Grafana v5.1 brings an improved official docker image which should make it easier to run and use the Grafana docker image and at the same time give more control to the user how to use/run it.
+
+We've switched the id of the grafana user running Grafana inside a docker container. Unfortunately this means that files created prior to 5.1 won't have the correct permissions for later versions and thereby this introduces a breaking change.
+We made this change so that it would be easier for you to control what user Grafana is executed as (see examples below).
+
+Version | User    | User ID
+--------|---------|---------
+< 5.1   | grafana | 104
+>= 5.1  | grafana | 472
+
+Please read the [updated documentation](/installation/docker/#migration-from-a-previous-version-of-the-docker-container-to-5-1-or-later) which includes migration instructions and more information.
+
+## Improved workflow for provisioned dashboards
+
+{{< docs-imagebox img="/img/docs/v51/provisioning_cannot_save_dashboard.png" max-width="800px" class="docs-image--right" >}}
+
+Grafana v5.1 brings an improved workflow for provisioned dashboards:
+
+* A populated `id` property in JSON is now automatically removed when provisioning dashboards.
+* When making changes to a provisioned dashboard you can `Save` the dashboard which now will bring up a *Cannot save provisioned dashboard* dialog like seen in the screenshot to the right.
+
+
+Available options in the dialog will let you `Copy JSON to Clipboard` and/or `Save JSON to file` which can help you synchronize your dashboard changes back to the provisioning source.
+More information in the [Provisioning documentation](/features/datasources/prometheus/).
+
+<div class="clearfix"></div>
+
+## New template variable interpolation syntax
+
+
+## Alerting
+
+New enhancements includes support for retries on failed alert queries and updating correct state when pausing/unpausing alerts.
+
+## Datasources
+
+### Prometheus
+
+New enhancements includes support for Prometheus histograms in the heatmap panel and autocomplete of template variables query editor. More information in the [Prometheus data source documentation](/features/datasources/prometheus/).
```

Comment:
I think support for Prometheus histograms should be highlighted more. Its a really big feature for that community.